docs/time/index.rst

```diff
@@ -1212,6 +1212,16 @@ jd         :class:`~astropy.time.TimeDeltaJD`
 datetime   :class:`~astropy.time.TimeDeltaDatetime`
 =========  ===================================================
 
+.. warning:: The ``datetime`` format does *not* support instants of time that
+             are leap seconds. This is a 
+             `limitation of the Python standard library datetime object <https://github.com/python/cpython/issues/67762>`_
+             not a limitation of Astropy's |Time|. If the upstream datetime
+             object is updated to support leap seconds Astropy can follow it,
+             but until then ``t.datetime`` will raise an exception if ``t`` is 
+             a leap second. See 
+             `astropy issue #14112 <https://github.com/astropy/astropy/issues/14112>`_ 
```

Comment:
I'm not sure that referencing #14112 is the best idea here, since that issue meanders a bit and starts with an (incorrect) statement that there is bug in astropy. After some discussion the issue gets to a better understanding but overall I think that the summary version here before this "See ..." sentence gets the essence of issue.